tvaultuiv2/src/views/private/safe/components/ListItem/index.js

```diff
@@ -34,23 +40,87 @@ const Flag = styled('span')`
   font-style: ${(props) => (props.fontStyle ? props.fontStyle : '')};
 `;
 
+const FolderIconWrap = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  .MuiSvgIcon-root {
+    width: 3rem;
+    height: 3rem;
+    :hover {
+      background: #151820;
```

Comment:
Hardcoded background.. Can we use it grom themes?

Reply:
added one field for icon hover effect color